Simple-Multiparty/build.gradle

```diff
@@ -15,9 +15,8 @@ buildscript {
 allprojects {
     repositories {
         jcenter()
-        maven {
-            url  "http://tokbox.bintray.com/maven"
-        }
+        maven { url 'https://maven.google.com' }
```

Comment:
@hbaqai  I think if you download the sdk tool version with the SDK manager, probably, you will not need to add the repo in the gradle file